<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>polygonToPolyLineDialog</class>
 <widget class="QDialog" name="polygonToPolyLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polygon To Polyline</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="Q3ButtonGroup" name="polygonToPolyLineGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>580</width>
     <height>221</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="Q3Table" name="inputOutputTable" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>500</width>
      <height>180</height>
     </rect>
    </property>
    <property name="lineWidth" stdset="0">
     <number>2</number>
    </property>
    <property name="midLineWidth" stdset="0">
     <number>0</number>
    </property>
    <property name="margin" stdset="0">
     <number>0</number>
    </property>
    <property name="dragAutoScroll" stdset="0">
     <bool>false</bool>
    </property>
    <property name="numRows" stdset="0">
     <number>0</number>
    </property>
    <property name="numCols" stdset="0">
     <number>2</number>
    </property>
    <property name="showGrid" stdset="0">
     <bool>true</bool>
    </property>
    <property name="rowMovingEnabled" stdset="0">
     <bool>false</bool>
    </property>
    <property name="columnMovingEnabled" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addFileButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>addFile</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="polygontopolylinedialog.qrc">
      <normaloff>:/icons/Plus32.png</normaloff>:/icons/Plus32.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFileButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>removeFile</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="polygontopolylinedialog.qrc">
      <normaloff>:/icons/Minus32.png</normaloff>:/icons/Minus32.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeAllButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>remove All Files</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="polygontopolylinedialog.qrc">
      <normaloff>:/icons/Delete32.png</normaloff>:/icons/Delete32.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="Q3TextBrowser" name="MessageLog" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>286</y>
     <width>580</width>
     <height>92</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>494</x>
     <y>240</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Run!</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>240</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends></extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="polygontopolylinedialog.qrc"/>
 </resources>
 <connections/>
</ui>
